src/lib/logical_query_plan/create_table_node.cpp

```diff
@@ -3,19 +3,18 @@
 #include <sstream>
 
 #include "constant_mappings.hpp"
+#include "static_table_node.hpp"
 
 namespace opossum {
 
-CreateTableNode::CreateTableNode(const std::string& table_name, const TableColumnDefinitions& column_definitions,
-                                 const bool if_not_exists)
-    : BaseNonQueryNode(LQPNodeType::CreateTable),
-      table_name(table_name),
-      column_definitions(column_definitions),
-      if_not_exists(if_not_exists) {}
+CreateTableNode::CreateTableNode(const std::string& table_name, const bool if_not_exists)
+    : BaseNonQueryNode(LQPNodeType::CreateTable), table_name(table_name), if_not_exists(if_not_exists) {}
 
 std::string CreateTableNode::description() const {
   std::ostringstream stream;
 
+  const auto column_definitions = std::dynamic_pointer_cast<StaticTableNode>(left_input())->table->column_definitions();
```

Comment:
and what if left_input is no StaticTableNode? It could be any LQP, right?

Reply:
Yes, you're right. So this means that the column_definitions may be still unknown. We could either leave out the column_definitions completely or insert the left_inputs description.